src/nft/pages/profile/sell.css.ts

```diff
@@ -22,7 +22,6 @@ export const mobileSellWrapper = style([
     zIndex: { sm: '3', md: 'auto' },
     height: { sm: 'full', md: 'auto' },
     width: 'full',
-    overflowY: 'scroll',
```

Comment:
double checking this removal is intentional and handled appropriately

Reply:
it's intentional.  There were multiple overflows that were showing a scrollbar when nothing was scrollable and it now only shows when content is actually scrollable